src/components/PaymentInfoLine.vue

```diff
@@ -63,14 +142,20 @@ function fiatAmountInfoValidator(value: any) {
         && typeof value.currency === 'string';
 }
 
-@Component({components: {Account, Timer, Amount, FiatAmount, ArrowRightSmallIcon}})
+@Component({components: {Account, Timer, Amount, FiatAmount, Tooltip, ArrowRightSmallIcon}})
 class PaymentInfoLine extends Vue {
-    private get originDomain() {
-        return this.origin.split('://')[1];
-    }
+    private static readonly REFERENCE_RATE_UPDATE_INTERVAL = 60000; // every minute
+    private static readonly RATE_DEVIATION_THRESHOLD = .05;
 
     @Prop({type: Object, required: true, validator: cryptoAmountInfoValidator}) public cryptoAmount!: CryptoAmountInfo;
     @Prop({type: Object, validator: fiatAmountInfoValidator}) public fiatAmount?: FiatAmountInfo;
+    // Note that merchantFee and networkFee have no effect if fiatAmount is not set, as the tooltip in which they appear
+    // is only visible when fiatAmount is set. As the fiatAmount was only introduced in the v2 checkout request in the
+    // hub, the tooltip and merchantFee and networkFee are thus never visible for v1 checkout requests. This is ok
+    // though as the merchantFee also only exists in v2 and the free-service-info doesn't make too much sense for
+    // nimiq.shop or the nimiq voting app which are currently the main apps using the v1 checkout.
```

Comment:
I would leave out the last sentence of the comment here. The tooltip is not displayed for non-fiat checkouts, period. Talking about other apps will not age well, and `This is ok though` is also an opinion, not a fact, so not very helpful I think.

**H**ub should be capitalized.

Reply:
I made minor adjustments to the comment but see the last sentence as essential as it gives the reasoning why this component does not show a tooltip for checkouts without fiat amount.